app/src/main/java/org/wikipedia/descriptions/DescriptionSuggestionService.kt

```diff
@@ -0,0 +1,18 @@
+package org.wikipedia.descriptions
+
+import io.reactivex.rxjava3.core.Observable
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface DescriptionSuggestionService {
+    @GET("article")
+    fun getSuggestion(
+        @Query("lang") lang: String,
+        @Query("title") title: String,
+        @Query("num_beams") count: Int
+    ): Observable<DescriptionSuggestionResponse>
```

Comment:
Would it be possible to use `conroutine` instead of `RxJava`?

Reply:
yes!